os/kernel/irq/irq.h

```diff
@@ -74,6 +74,9 @@
 struct irq {
 	xcpt_t handler;
 	FAR void *arg;
+#ifdef CONFIG_ENABLE_IRQINFO
+	size_t count;
```

Comment:
How about bind this with some debug config? We are in progress to reduce memory usages but it spends memory defaultly. Or if it is needed only for debug, let's make a dependancy with DEBUG config.